react-delegationdashboard/src/pages/Dashboard/Actions/DelegateAction/DelegateModal.tsx

```diff
@@ -52,17 +52,17 @@ const DelegateModal = ({ show, balance, handleClose, handleContinue }: DelegateM
           )}
           <Formik
             initialValues={{
-              amount: '10',
+              amount: '1',
```

Comment:
This value can be configured.
Another option would be to take it from SC:
`{
    "scAddress": "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqylllslmq6y6",
    "funcName": "getContractConfig"
}`

Returns the followin as a list: 
`NumContracts - total number of contracts deployed by the manager (big int)
LastAddress - last generated delegation SC address (base64)
MinServiceFee (big int)
MaxServiceFee (big int)
MinDeposit - minimum deposit when creating a new delegation SC (big int)
MinDelegationAmount - minimum value to be delegated (big int)`

Reply:
I will fix this in another PR :)